SPECS/polkit/polkit.spec

```diff
@@ -40,6 +40,9 @@ header files and libraries for polkit
 
 %prep
 %autosetup -p1
+# Disable polkitbackend tests, which fail since dbus is not available in the worker chroot
```

Comment:
Would adding it as a `BuildRequires` gated by `%if %{with_check}` work?

Reply:
I've tried this, but even with our dbus package installed the test produces the following error:

"/PolkitBackendJsAuthority/get_admin_identities: Error getting system bus: Could not connect: No such file or directoryError getting login monitor: -2"

Reply:
Intereseting. Maybe dbus doesn't just work well inside the chroot. We should probably investigate at some point, but that's outside the scope of this change, I think.